demos/ws-bootable-jar-example/jaxws/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2014, Red Hat, Inc., and individual contributors
+  ~ as indicated by the @author tags. See the copyright.txt file in the
+  ~ distribution for a full listing of individual contributors.
+  ~
+  ~ This is free software; you can redistribute it and/or modify it
+  ~ under the terms of the GNU Lesser General Public License as
+  ~ published by the Free Software Foundation; either version 2.1 of
+  ~ the License, or (at your option) any later version.
+  ~
+  ~ This software is distributed in the hope that it will be useful,
+  ~ but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~ Lesser General Public License for more details.
+  ~
+  ~ You should have received a copy of the GNU Lesser General Public
+  ~ License along with this software; if not, write to the Free
+  ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.jboss.intersmash</groupId>
+        <artifactId>ws-bootable-jar-example</artifactId>
+        <version>0.0.1-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>jaxws</artifactId>
+    <packaging>war</packaging>
+
+    <name>SOAP-example</name>
```

Comment:
s/SOAP-example/Intersmash demos :: WildFly Webservices Bootable JAR Examples :: SOAP Example/

Reply:
Can we please set this value to `SOAP-example/Intersmash demos :: WildFly Webservices Bootable JAR Examples :: SOAP Example`? 
It needs to be consistent with the other modules in the project, in order to provide a better output, e.g.:

```
INFO] Intersmash Shared Deployments (Wildfly): WildFly Bootable JAR (javax version) SUCCESS [  0.036 s]
[INFO] Intersmash Shared Deployments (Wildfly): Hello World example SUCCESS [  0.050 s]
[INFO] Intersmash Deployments: Deployments Provider ....... SUCCESS [  0.083 s]
[INFO] Intersmash Standalone Deployments: OpenShift Jakarta Sample SUCCESS [  0.028 s]
[INFO] Intersmash Testsuite ............................... SUCCESS [  0.421 s]
[INFO] intersmash-demos ................................... SUCCESS [  0.006 s]
[INFO] ws-bootable-jar-example ............................ SUCCESS [  0.192 s]
[INFO] SOAP-example ....................................... SUCCESS [  0.990 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------

```
As you can see the last 3 modules have a different description, which should be improved by setting those `<name>` tags in their POM.

Reply:
For consistency and clarity I changed the demos module names following 
the  (Intersmash Shared Deployments:) naming convention.  The names now
are this.  

[INFO] Intersmash Demos 
[INFO] Intersmash Demos : (Wildfly): Webservices Bootable JAR Examples 
[INFO] Intersmash Demos : (Wildfly): Webservices Bootable JAR Examples (jaxws)